### ISSUE TITLE
[PageControl] Minimum height should be 48pt to meet accessibility requirements

### ISSUE THREAD

**Issue by jverkoey** [material-components/material-components-ios#4287]
The default height for MDCPageControl is [37](https://github.com/material-components/material-components-ios/blob/develop/components/PageControl/src/MDCPageControl.m#L33). Accessibility guidelines require that any interactive control be at least 48 points tall.

## Definition of done

- [ ] The page control's minimum height is 48 points.
